docs/3-cloud-computing/3.2.6-eks.md

```diff
@@ -125,6 +158,18 @@ Finally we will add an Ingress controller so that you can access this cluster wi
 3. Run `kubectl get Ingress -n sock-shop` in order to get the address of your Ingress.
 4. Make sure that your cluster works the same way that it would when you were port-forwarding it.
 
+## Exercise - Create a LoadBalancer Service for (AWS Cluster)
+
+1. Use the same namespace and pod from the ClusterIP exercise.
+
+2. Create a service of type LoadBalancer. Folow the same guidelines as the previous service, remove the nodePort.
+
+3. Create an ingress that uses the external ip from your LoadBalancer service as the host on port `8080`. It should use the path `/` and the jenkins service name.
```

Comment:
I think this is kind of confusing because you don't really need the ingress here. At this point you can directly access Jenkins using the LoadBalancer service and load balancer hostname. I think it would be good to move this before the ingress controller exercise and just focus this on how the LoadBalancer services works.